app/controllers/foods_controller.rb

```diff
@@ -1,22 +1,32 @@
 class FoodsController < ApplicationController
   def index
+    @food = Food.all
+  end
 
+  def show
+    @food = Food.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
+    flash[:alert] = "The food you were looking for could not be found."
+    redirect_to foods_path
   end
+
   def new
-    @food = Food.new(food_params)
+    @food = Food.new
   end
-def create
-  @food = Food.create(food_params)
-  if food.save
-    redirect_to foods_path, notice: "Food create successfully"
-  else
-    flash["alert"] = "Food not created"
-end
+
+  def create
+    @food = Food.new(food_params)
```

Comment:
What do you think about rewriting this as
`@food = current_user.foods.new(food_params)`